README.md

```diff
@@ -1 +1,11 @@
-# Ali-Edly-Arbisoft-Training
\ No newline at end of file
+# Django-React-Blog-App
+
+Installation Process:
+1. Clone this repository onto your system
+2. Change directory from root directory to `/webblog/`
+3. Open terminal and start virtual environment by running `source ../env/bin/activate`
```

Comment:
We should never commit env folder or private configuration folders/files. Add additional steps to create virtualenv from scratch and installing requirements.